test/Microsoft.Extensions.Configuration.FunctionalTests/ConfigurationTests.cs

```diff
@@ -680,7 +680,9 @@ public void CanSetValuesAndReloadValues()
         }
 
 
-        [Theory]
+        [ConditionalTheory]
+        // Disabling until https://github.com/aspnet/Configuration/issues/628 is resolved.
+        [OSSkipCondition(OperatingSystems.MacOSX)]
```

Comment:
Again, pleas use `[ConditionalTheory(Skip = "reason and issue url")]` or similar when disabling tests